code/src/styles.js

```diff
@@ -0,0 +1,96 @@
+const styles = {
```

Comment:
How come you did the styling this way instead of in "normal" css?

Reply:
See comment under Radium question